README.md

```diff
@@ -20,6 +20,7 @@ Available addons
 addon | version | summary
```

Comment:
Please format the README file following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md